pages/common/git-branch.md

```diff
@@ -10,6 +10,10 @@
 
 `git branch -a`
 
+- List unmerged branches (local):
```

Comment:
Is there a need for this ? You can figure this out from `git branch` and `git branch -a` commands that which branches are unmerged.

Reply:
Is this because merged branches are deleted in the remote? Personally I haven't adopted that workflow yet so finding out about ``git branched --no-merged`` was very useful

Reply:
Oh you mean remote branches which are merged with master ? Yes, then you need this command. But I do not find this to be useful to beginners getting started with git; which this page is aimed at.

Getting a list of local and remote branches is a good place to start.

Reply:
I also mean local branches that are merged with master. but I can see what you mean with this not being the most beginner-friendly. It's just a neat option I found that I thought would be useful. Closing the PR

Reply:
I don't work with branches often, but this sounds useful if you're unsure if a branch actually has been merged with master or not?